app/models/setting.rb

```diff
@@ -71,7 +72,12 @@ def self.config_file
   end
 
   def self.live_descendants
-    self.disabled_plugins.default_organization.organization_fact.default_location.location_fact
+    self.disabled_plugins.default_organization.organization_fact.default_location.location_fact.order_by(:category)
+  end
+
+  def self.stick_general_first
+    sticky_setting = 'Setting::General'
+    where(:category => sticky_setting).concat(where.not(:category => sticky_setting)).group_by(&:category)
```

Comment:
The issue mentions the rest of the settings should be ordered alphabetically, where is that?